src/client/java/hadoop-daos/src/test/java/io/daos/fs/hadoop/DaosFileSystemContractIT.java

```diff
@@ -207,15 +120,17 @@ public void testGetFileStatusFileAndDirectory() throws Exception {
 
   @Test
   public void testMkdirsForExistingFile() throws Exception {
+    try{
     Path testFile = this.path("/test/hadoop/file");
     assertFalse(this.fs.exists(testFile));
-    this.fs.mkdirs(new Path("/test/hadoop"));
     this.createFile(testFile);
     assertTrue(this.fs.exists(testFile));
     this.fs.mkdirs(testFile);
+//    fail("/test/hadoop/file is a file");
+    }catch (FileAlreadyExistsException e){
```

Comment:
please add space before/after {}

Reply:
we usually don't check the styling in test code. But as you pointed out, I'll check them all. thanks.

Reply:
done